docs/source/user/urls.rst

```diff
@@ -25,23 +25,22 @@ Entering this URL will open the notebook in JupyterLab in
 :ref:`single-document mode <tabs>`.
 
 
-.. _url-workspaces:
+.. _url-workspaces-ui:
 
-Managing Workspaces
-~~~~~~~~~~~~~~~~~~~
+Managing Workspaces (UI)
+~~~~~~~~~~~~~~~~~~~~~~~~
 
 JupyterLab sessions always reside in a workspace. Workspaces contain the state
 of JupyterLab: the files that are currently open, the layout of the application
 areas and tabs, etc. When the page is refreshed, the workspace is restored.
 
-The default workspace is not named and only saves its state on the
-user's local browser:
+The default workspace is not named, it is the primary ``/lab`` URL.
```

Comment:
"it resides at the primary ``/lab`` URL." ? or "is accessed via the primary ``/lab`` URL."? Also would it be better to start with "The default workspace does not have a name"? "...is not named" sounds like maybe you could name it, but it just isn't named by default.

Reply:
Cheers, @ivanov, will update it in the morning. Hopefully CI will be happy then, too.